payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressSpec.kt

```diff
@@ -5,23 +5,22 @@ import com.stripe.android.ui.core.R
 import com.stripe.android.ui.core.address.AddressFieldElementRepository
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
 
 @Serializable
 enum class DisplayField {
     @SerialName("country")
     Country
 }
 
-@Serializable
-enum class AddressType {
-    @SerialName("shipping_condensed")
-    ShippingCondensed,
-
-    @SerialName("shipping_expanded")
-    ShippingExpanded,
-
-    @SerialName("normal")
-    Normal
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
```

Comment:
what's the run down on this having to be serialized?

Reply:
@michelleb-stripe feel free to chime in here, but the tl;dr is that we are ok with some fields not needing to be deserialized when needed. The main concern is that maybe we want a future where you can create a form without form spec, and use elements directly.